### ISSUE TITLE
small graphical details between LWC desktop mode and mobile mode

### ISSUE THREAD

**Issue by josemvm** [3liz/lizmap-web-client#701]
hi,

I found some small graphical details:

**mobile mode:**

- the "Login" header truncates and overlaps the logo:
![image1](https://user-images.githubusercontent.com/10053874/32406325-5c83e580-c16e-11e7-854a-a9685bfe412f.jpg)
-  the "Administration" header overlaps the logo and once inside the administration menu, the strings for the logged user/"admin" and "Projects" have a different char font:
![image2](https://user-images.githubusercontent.com/10053874/32406326-5c9e16a8-c16e-11e7-8051-338c407f34d1.jpg)
- there's an empty space on left in the headers (Administration/image above, Main and Projects)
![image3](https://user-images.githubusercontent.com/10053874/32406327-5cbb2216-c16e-11e7-9426-5ca5320af3a0.jpg)
![proj](https://user-images.githubusercontent.com/10053874/32406324-5c66cf22-c16e-11e7-8457-f891ed768c33.jpg)

inside the map project the header it's ok:
![ok](https://user-images.githubusercontent.com/10053874/32406494-791578f0-c171-11e7-9d9b-f2dc3ddd48bb.jpg)

**desktop mode:**
- after login, drop down menu - behavior ok:
![login1](https://user-images.githubusercontent.com/10053874/32406750-2d560e3e-c176-11e7-9b84-dc9e5257aaab.jpg)
- after login, inside the Administration menu/My account - different char font:
![logadmi](https://user-images.githubusercontent.com/10053874/32406914-61104030-c178-11e7-9adc-a385e04abd8c.jpg)
-  **(master branch, after commit https://github.com/3liz/lizmap-web-client/commit/b83f1a59f119b8804e87b3ee5d0fbd10d2648dde)** inside the map project, after login - this behavior, 
![lll](https://user-images.githubusercontent.com/10053874/32407010-e23c30be-c179-11e7-990b-46912a02f527.jpg)

thanks in advance for your attention

**Comment by rldhont** [comment]
Thanks @josemvm if you have some hack or propositions to fix them, thanks in advance.

**Comment by mdouchin** [comment]
@josemvm @rldhont I just pushed a commit, not related to this, but concerning CSS. I was a bit annoyed by all the bold fonts in Lizmap UI. I removed many of them.
Do you like it ? I really do...

**Comment by josemvm** [comment]
hi @mdouchin 
yes, that seems good to me!

**Comment by rldhont** [comment]
The bold is in the template:
```html

      <li class="home">
     
     <a href="/~dhont/lizmap/lizmap/www/index.php/view/" title="Projets">
       <span class="icon"></span>
       <span class="text"><b>Projets</b></span>
     </a>
     
    </li>
    <li class="user dropdown">
    <a class="dropdown-toggle" data-toggle="dropdown" href="#" id="info-user">
      <span class="icon"></span>
      <span class="text"><b id="info-user-login">admin</b></span>
      <span class="caret"></span>
    </a>
    <ul class="dropdown-menu pull-right">
      
      <li><a href="/~dhont/lizmap/lizmap/www/admin.php/jauthdb_admin/user/?j_user_login=admin">Mon compte</a></li>
            <li><a href="/~dhont/lizmap/lizmap/www/admin.php/jauth/login/out" id="info-user-logout">Déconnexion</a></li>
    </ul>
  </li>

```
@mdouchin and @josemvm do you prefer to remove it ?

**Comment by rldhont** [comment]
@josemvm the idea for the dropdown is to have which color ?
The one of the home/admin page or of the map ?

**Comment by josemvm** [comment]
hi @rldhont 

the idea is the same font size and type, not the color
![imagem](https://user-images.githubusercontent.com/10053874/32506530-b6e28c50-c3dc-11e7-8afe-bebb13b9cda3.png)

**Comment by rldhont** [comment]
To fix this, and have something homogeneous between admin and home, the admin template has to be updated to have the same html organization.

In home:
```html
<div id="header">
  <div id="logo"></div>
  <div id="title"></div>
  <div id="headermenu"></div>
</div>
```

In admin:
```html
<div id="header">
  <div id="logo"></div>
</div>
<div id="headermenu"></div>
```

**Comment by rldhont** [comment]
the empty space on left in the headers is a padding-right defined in media.css when the screen has an height lower than 640px:
```css
only screen and (-webkit-min-device-pixel-ratio: 2),
only screen and (   min--moz-device-pixel-ratio: 2),
only screen and (     -o-min-device-pixel-ratio: 2/1),
only screen and (        min-device-pixel-ratio: 2),
only screen and (                min-resolution: 192dpi),
only screen and (                min-resolution: 2dppx),
only screen and (                   max-height : 640px) ,
only screen and (                   max-device-height : 640px)
```

**Comment by josemvm** [comment]
and for this?

image:
![imagem](https://user-images.githubusercontent.com/10053874/32506997-17a7dcba-c3de-11e7-9c5e-2d3ce923ff1f.png)

**Comment by rldhont** [comment]
probably the same as the dropdown -> HTML marker organization.

**Comment by josemvm** [comment]
thanks @rldhont

**Comment by rldhont** [comment]
@josemvm nada

**Comment by josemvm** [comment]
"de nada" :-)

**Comment by josemvm** [comment]
just one last small detail (**mobile mode**):

![imagem](https://user-images.githubusercontent.com/10053874/32631400-0d8baa0c-c598-11e7-9e6b-c27eb01b5fb4.png)


strings "Connect" and "Register" are missing. we can only see the icons...

**Comment by rldhont** [comment]
@josemvm do you have an idea to fixe on mobile: strings "Connect" and "Register" are missing. we can only see the icons...

**Comment by josemvm** [comment]
@rldhont, i have no idea how to fix it

**Comment by rldhont** [comment]
@josemvm fixed by @SenequeC with 96c2df7fde4d67cdcb4b5aa614ce91f4235c32ed

**Comment by josemvm** [comment]
the new icon doesn't appear here...
